CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ## [UNRELEASED] - XXXX-XX-XX
+### Added
+- Change the default cursor to a spinning wheel on linux when the data is changed ([#341](https://github.com/cbrnr/mnelab/pull/341) by [Guillaume Favelier](https://github.com/GuillaumeFavelier))
```

Comment:
```suggestion
- Change the default cursor to a spinning wheel when the data is changed ([#341](https://github.com/cbrnr/mnelab/pull/341) by [Guillaume Favelier](https://github.com/GuillaumeFavelier))
```

Reply:
No need to mention Linux (mostly because people seeing the code might wonder why you didn't mention Windows).